### PR TITLE
Gp2 210 personalisation bar restyle

### PR DESCRIPTION
**Pull request by bobmeredith**
To do (delete all that do not apply):

 - [Y] Change has a jira ticket that has the correct status.
 - [Y] Changelog entry added.
 - [N] I have updated security dependencies

### CODE REVIEW

File: learn/templates/learn/detail_page.html [hunk 1]
```diff
@@ -32,84 +32,28 @@ <h3 class="h-s">{{ topic.title }}</h3>
...
-                            {% endfor %}
-                        {% else %}
-                            <span class="great-mvp-pill-button">add countries</span>
+                {% include "learn/includes/personalisation_bar.html" with profile=request.user.company %}
+            </div>
+            <div class="grid p-t-l p-f-l">
+                <div class="c-2-3">
+                    <h1 class="h-l" id="lesson-title">{{ self.title }}</h1>
+                    {% include 'core/personalised_body.html' %}
+                    <div id="countries-chooser" class="great-mvp-full-width-background"></div>
```
Comment (lewis-coulson-dit) [inline learn/templates/learn/detail_page.html:41]: I think we want to start removing any class names with an `mvp` in them now so change `great-mvp-full-width-background` to `great-full-width-background`

File: learn/templates/learn/includes/personalisation_bar.html [hunk 2]
```diff
@@ -0,0 +1,52 @@
+
+{% load wagtailimages_tags %}
+{% load routablepageurl from wagtailroutablepage_tags %}
+
+<nav class="p-t-xs p-h-xs p-b-xxs great-mvp-full-width-background" id="market-select-navbar">
```
Comment (lewis-coulson-dit) [inline learn/templates/learn/includes/personalisation_bar.html:5]: Same here

File: learn/templates/learn/detail_page.html (hunk 1 above)
Comment (bobmeredith) [reply learn/templates/learn/detail_page.html:41]: Fair enough. However, I don't see the point of leaving styles like this in great-cms. This looks like a candidate for a helper class.  I can't find one that quite does the job but I could slightly modify one.

Comment (bobmeredith) [reply learn/templates/learn/detail_page.html]: Added class w-full-viewport to great styles and deleted the old mvp style.
They'll be a lot more of this to do!

**Pull request merged**